src/generator.rs

```diff
@@ -215,7 +251,7 @@ safe_drive = {safe_drive_dep}
                             if let Some(stem) = path.file_stem() {
                                 let idl_file = format!("{}.idl", stem.to_str().unwrap());
                                 let idl_file = path.parent().unwrap().join(idl_file);
-                                if !idl_file.exists() {
+                                if !cfg!(target_os = "windows") && !idl_file.exists() {
```

Comment:
on windows this is causing issue with .msg in srv folder. may be it can be fixed in another way